From 78a7fde6b1522bc54a6d0d358a4396acaf32d023 Mon Sep 17 00:00:00 2001
From: Damillora <developer@damillora.com>
Date: Fri, 27 Feb 2026 14:57:27 +0700
Subject: [PATCH 4/4] Remove redundant application name

---
 opencloud.desktop.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/opencloud.desktop.in b/opencloud.desktop.in
index 4320158f5..c42071c4d 100644
--- a/opencloud.desktop.in
+++ b/opencloud.desktop.in
@@ -2,8 +2,8 @@
 Categories=Utility;X-SuSE-SyncUtility;
 Type=Application
 Exec=@APPLICATION_EXECUTABLE@ --showsettings
-Name=@APPLICATION_NAME@ desktop sync client
-Comment=@APPLICATION_NAME@ desktop synchronization client
+Name=@APPLICATION_NAME@
+Comment=@APPLICATION_NAME@ synchronization client
 GenericName=Folder Sync
 Icon=@APPLICATION_REV_DOMAIN@
 Keywords=@APPLICATION_NAME@;syncing;file;sharing;
-- 
2.52.0

